packages/xo-web/src/xo-app/pool/tab-advanced.js

```diff
@@ -39,56 +47,129 @@ class PoolMaster extends Component {
+    )
+
+  render () {
+    const { hosts, gpuGroups, pool } = this.props
+    const { editRemoteSyslog } = this.state
+    return (
+      <div>
+        <Container>
+          <Row>
+            <Col>
+              <h3>{_('xenSettingsLabel')}</h3>
+              <table className='table'>
+                <tbody>
+                  <tr>
+                    <th>{_('uuid')}</th>
+                    <Copiable tagName='td'>{pool.uuid}</Copiable>
+                  </tr>
+                  <tr>
+                    <th>{_('poolHaStatus')}</th>
+                    <td>
+                      {pool.HA_enabled
+                        ? _('poolHaEnabled')
+                        : _('poolHaDisabled')}
+                    </td>
+                  </tr>
+                  <tr>
+                    <th>{_('setpoolMaster')}</th>
+                    <td>
+                      <PoolMaster pool={pool} />
+                    </td>
+                  </tr>
+                  <tr>
+                    <th>{_('syslogRemoteHost')}</th>
+                    <td>
+                      <ul className='pl-0'>
+                        {map(hosts, host => (
+                          <li key={host.id}>
+                            <span>{`${host.name_label}: `}</span>
+                            <Text
+                              value={host.logging.syslog_destination || ''}
+                              onChange={value =>
+                                setRemoteSyslogHost(host, value)
+                              }
+                            />
+                          </li>
+                        ))}
+                      </ul>
+                      <ActionRowButton
+                        btnStyle={editRemoteSyslog ? 'info' : 'primary'}
+                        handler={this.toggleState('editRemoteSyslog')}
+                        icon='edit'
+                      >
+                        {_('poolEditAll')}
+                      </ActionRowButton>
+                      {editRemoteSyslog && (
+                        <form
+                          id='formRemoteSyslog'
+                          className='form-inline mt-1'
+                        >
+                          <div className='form-group'>
+                            <input
+                              className='form-control'
+                              onChange={this.linkState('syslogDestination')}
+                              placeholder={this.props.intl.formatMessage(
+                                messages.poolRemoteSyslogPlaceHolder
+                              )}
+                              type='text'
+                            />
+                          </div>
+                          <div className='form-group ml-1'>
+                            <ActionButton
+                              btnStyle='primary'
+                              data-syslogDestination={
+                                this.state.syslogDestination
```

Comment:
You don't need to pass it via the `data-*`. `_setRemoteSyslogHosts` can access the `state` by itself.